site/src/docs/components/highlight/code.mdx

```diff
@@ -0,0 +1,146 @@
+---
+slug: '/components/highlight/code'
+title: 'Highlight - Code'
+---
+
+import { Button, Highlight, Link } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import Playground from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Code
+
+### Code examples
+
+#### Highlight and quote sizes
+
+<Playground>
+
+
+```jsx
+<>
+  <Highlight text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to." variant="l"/>
+  <Highlight text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to."/>
+  <Highlight text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to." variant="s"/>
+  <Highlight text="Add an interesting quote here" variant="l" type="quote" reference="First name Last name. Title."/>
+  <Highlight text="Add an interesting quote here" type="quote" reference="First name Last name. Title."/>
+  <Highlight text="Add an interesting quote here" variant="s" type="quote" reference="First name Last name. Title."/>
```

Comment:
The styles seem to be different for React and Core, why is that? 😮

Reply:
class for blockquote tag was missing from core  examples.